style.css

```diff
@@ -0,0 +1,171 @@
+@media (min-width: 250px) {
+  * {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
+    background: #1c1a19;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  }
```

Comment:
- [ ] [Optinal] You should use the font `font-family: 'Crete Round';` please implement it